pkg/apis/certmanager/v1alpha1/types_issuer.go

```diff
@@ -274,6 +275,19 @@ type ACMEIssuerDNS01ProviderRFC2136 struct {
 	TSIGAlgorithm string `json:"tsigAlgorithm"`
 }
 
+type ACMEIssuerDNS01ProviderWebhook struct {
+	URL string `json:"url"`
+
+	// + optional
+	Metadata map[string]string `json:"metadata"`
+
+	// +optional
+	SkipTLSVerify bool `json:"skipTLSVerify"`
+
+	// +optional
+	WebhookCASecret SecretKeySelector `json:"webhookCASecretSecretRef"`
```

Comment:
I think we should probably call this field `caSecretRef` to keep it shorter (it is already clear that this is for the webhook given the struct name).

I'm also unsure whether this should be a secret reference, or a `caBundle` field like in other Kubernetes resource types. Perhaps we should support both, although for now I'm happy to keep it as-is and look at adding `caBundle` later 😄